src/index.html

```diff
@@ -146,38 +146,38 @@ <h3>STAIRS TODAY</h3>
           </section>
         </section>
         <section class='info-card hide' id='stairs-info-card'>
-          <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
+          <!-- <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
           <section class='card-data-line'>
             <p>FLIGHTS CLIMBED TODAY</p>
             <h4 id='stairs-info-flights-today'></h4>
-          </section>
+          </section> -->
         </section>
         <section class='friends-card hide' id='stairs-friends-card'>
-          <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
+          <!-- <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
           <section class='card-data-line'>
             <p>ALL USERS' AVERAGE FLIGHTS TODAY</p>
             <h4 id='stairs-friend-flights-average-today'></h4>
-          </section>
+          </section> -->
         </section>
         <section class='trending-card hide' id='stairs-trending-card'>
-          <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
+          <!-- <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
           <section class='card-data-line trending-stairs-phrase-container'>
-          </section>
+          </section> -->
         </section>
         <section class='calendar-card hide' id='stairs-calendar-card'>
-          <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
+          <!-- <button aria-label='go-back' class='go-back-button stairs-go-back-button'></button>
           <section class='card-data-line'>
```

Comment:
We can delete the commented-out sections in the next branch, but it's good that we have them in this PR in case we need to refer back to it.